### ISSUE TITLE
Caledor Chapter 22 Public Quest - No Rewards at Conclusion for All Players

### ISSUE THREAD

**Issue by aloghides** [WarEmu/WarBugs#16935]
**Caledor Chapter 22 PQ** - Zero members of the warband received rolls for PQ loot at the end when the final boss is killed.  Our warband attempted this PQ twice today 5/23/2020 and did not receive any reward both times.

**Notes:**
All members were in the PQ.  
20-21 players in the warband.
Some members were not level 40, but level 38 or so.

**Reproduction Steps:**
Complete the PQ as normal through all 3 stages with your warband.

**Screenshot:**
At the conclusion of the PQ, everyone simply sees the Resetting icon.  No windows popup to display rolls for loot/contribution.  Though, a chest does appear which I did not manage to get in the screenshot in time.
![chp22pq](https://user-images.githubusercontent.com/6305141/82739108-9136bb00-9d02-11ea-8d0b-278269ba61b9.png)

**Comment by bosli42** [comment]
Did your warband wipe or players disconnect?
For destro, if you disconnect / relog you lose all contribution. If you do it after stage 1, you won't get bags anymore.
Something similar happens if you die on the endboss and release on the kill-try.

**Comment by aloghides** [comment]
Yes we wiped once or twice and one player may have disconnected if memory serves me correctly.  The whole warband did not receive a reward.

**Comment by aloghides** [comment]
Another attempt today and got same issue

**Comment by aloghides** [comment]
Okay - this happened to several players in our CW warband.  Only to those who died and respawned at chp22 camp.  

**I think we have isolated the problem to any player who releases and respawns outside the PQ will not get a loot roll at the end of the PQ even if they come back and contribute.**
